### PR TITLE
Documentation of new groupings of individuals

### PR DESCRIPTION
**Pull request by MImmesberger**
- Added new grouping to GEP 1.
- Translated the hh_concepts document to english and edited it slightly to represent the current state of development.

As I presume that this document will eventually be replaced by some tutorial or proper documentation, I didn't put too much effort in making the document appealing for new users. LMK if you think that's necessary though.

### CODE REVIEW

File: docs/geps/gep-01.md [hunk 1]
```diff
@@ -119,8 +119,8 @@ If a column has a reference to a time unit (i.e., any flow variable like earning
 transfers), a column is indicated by an underscore plus one of {`y`, `m`, `w`, `d`}.
 
 The default unit a column refers to is an individual. In case of groupings of
-individuals, an underscore plus one of {`sn`, `hh`, `fg`, `bg`} will indicate the level
-of aggregation.
+individuals, an underscore plus one of {`sn`, `hh`, `fg`, `bg`, `ehe`, `eg`} will
```
Comment (hmgaudecker) [inline docs/geps/gep-01.md:122]:
```suggestion
individuals, an underscore plus one of {`sn`, `hh`, `fg`, `bg`, `eg`, `ehe`} will
```

File: docs/gettsim_developer/hh_concepts.md [hunk 2]
```diff
@@ -0,0 +1,263 @@
...
+## Kindergeld
+
+#### Description
+
+- Only one parent receives child allowances (specified via `p_id_kindergeld_empf`)
+- Relevant for Unterhaltzahlung of the other parent
+
+#### Relevant aggregation unit
+
+- `p_id` (exogenous)
```
Comment (hmgaudecker) [inline docs/gettsim_developer/hh_concepts.md:45]: Make clear this is a pointer and not an aggregation unit.

**Review by hmgaudecker — COMMENTED** [review]
Thanks!

File: docs/gettsim_developer/hh_concepts.md [hunk 3]
```diff
@@ -0,0 +1,263 @@
...
+
+- `bg_id` (endogenous)
+
+### SGB XII (Hilfe zum Lebensunterhalt)
+
+#### Description
+
+Government expenditures: 1.5 Mrd €
+
+- Relevant are
```
[the remark this answers is not in the document]
Comment (MImmesberger) [reply docs/gettsim_developer/hh_concepts.md:141]: I think Bedarf is on individual level and income/wealth checks are on Einstandsgemeinschaft (in the sense of SGB XII, hence, including children) level. 

Currently, we use the Einstandsgemeinschaft term of SGB II here (which should be wrong).

**Comment by MImmesberger** [comment]
I incorporated most of your comments. For each tax and transfer, we now have two or three Subheadings: "Description", "Aggregation unit" and "Pointers". Aggregation unit is only present if there is aggregation to the unit level. Pointers specify which pointers are used to i) create endogenous groups or ii) establish relationship between individuals (e.g. parent-child links for Kindergeld).

Still have to look at the Wohngeld comment.

File: docs/gettsim_developer/hh_concepts.md [hunk 4]
```diff
@@ -0,0 +1,346 @@
...
+
+#### Description
+
+- Children that are part of the Bedarfsgemeinschaft of their parents can receive
+  Wohngeld to drop out of the Bedarfsgemeinschaft ("Kinderwohngeld")
+- Only possible if the child can cover its needs with the Wohngeld transfer
+- Does not happen automatically, but upon request by the parents
+- Not implemented yet.
+
+## Modelling groupings
```
Comment (hmgaudecker) [inline docs/gettsim_developer/hh_concepts.md:307]: I am not sure about this section. Maybe label as some random notes?

**Review by hmgaudecker — APPROVED** [review]
Cool, thanks! Just a couple of small items for now, more important stuff to do in the next few weeks.

File: docs/gettsim_developer/hh_concepts.md (hunk 4 above)
Comment (MImmesberger) [reply docs/gettsim_developer/hh_concepts.md]: I changed the title to "Limitations" and tailored the content a bit.

**Pull request merged**